cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -175,6 +190,11 @@ public void execute() {
             sticky = false;
         }
 
+        if (enableNativeImage) {
+            this.outStream.println(
+                    "WARNING : Native image generation is an experimental feature, which supports only a limited set of functionality.");
```

Comment:
@azinneera  should we mention the fact that the functionality could change in further releases?